tests/test_candidate_routes.py

```diff
@@ -1,39 +1,93 @@
-import requests
 import setup
 import time
+import pytest
 
 from sqs_service import *
 import constants
+from conftest import SERVICE_MAP
 
 SQS_STAGE = "https://sqs.stage.osmosis.zone"
```

Comment:
SQS_STAGE and ROUTES_URL also exist in sqs_service.py. Shall we not duplicate them?

Reply:
Good catch - I think we will need to add a Python linter

Reply:
https://github.com/osmosis-labs/sqs/issues/new

Reply:
Whoops, I mislinked 😅  - this one: https://github.com/osmosis-labs/sqs/issues/220